src/lightning_fabric/README.md

```diff
@@ -1 +1,167 @@
+          input, target = batch
+-         input, target = input.to(device), target.to(device)
+          optimizer.zero_grad()
+          output = model(input)
+          loss = loss_fn(output, target)
+-         loss.backward()
++         fabric.backward(loss)
+          optimizer.step()
+          lr_scheduler.step()
+```
+
+______________________________________________________________________
+
+# Getting started
+
+## Install Lightning
+
+<details>
+
+<summary>Prerequisites</summary>
+
+> TIP: We strongly recommend creating a virtual environment first.
+> Don’t know what this is? Follow our [beginner guide here](https://lightning.ai/docs/stable/install/installation.html).
+
+- Python 3.8.x or later (3.8.x, 3.9.x, 3.10.x, ...)
+
+</details>
+
+```bash
+pip install -U lightning
+```
+
+## Convert your PyTorch to Fabric
+
+1. Create the `Fabric` object at the beginning of your training code.
+
+```
+import Lightning as L
+
+fabric = L.Fabric()
+```
+
+2. Call `setup()` on each model and optimizer pair and `setup_dataloaders()` on all your data loaders.
+
+```
+model, optimizer = fabric.setup(model, optimizer)
+dataloader = fabric.setup_dataloaders(dataloader)
+```
+
+3. Remove all `.to` and `.cuda` calls -> Fabric will take care of it.
+
+```diff
+- model.to(device)
+- batch.to(device)
+```
+
+4. Replace `loss.backward()` by `fabric.backward(loss)`.
+
+```diff
+- loss.backward()
++ fabric.backward(loss)
+```
+
+4. Run the script from the terminal with
+
+```bash
+lightning run model path/to/train.py
+```
+
+or use the launch() method in a notebook. Learn more about [launching distributed training](https://pytorch-lightning.readthedocs.io/en/stable/fabric/fundamentals/launch.html).
+
+______________________________________________________________________
+
+# FAQ
+
+## When to use Fabric?
+
+- **Minimum code changes**- You want to scale your PyTorch model to use multi-GPU or use advanced strategies like DeepSpeed without having to refactor. You don’t care about structuring your code- you just want to scale it as fast as possible.
+- **Maxmium control**- Write your own training and/or inference logic down to the individual optimizer calls. You aren’t forced to conform to a standardized epoch-based training loop like the one in Lightning Trainer. You can do flexible iteration based training, meta-learning, cross-validation and other types of optimization algorithms without digging into framework internals. This also makes it super easy to adopt Fabric in existing PyTorch projects to speed-up and scale your models without the compromise on large refactors. Just remember: With great power comes a great responsibility.
+- **Maxmium flexibility**- You want to have full control over your entire training- in Fabric all features are opt-in, and it provides you with a tool box of primitives so you can build your own Trainer.
+
+## When to use the [Lightning Trainer](https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html)?
```

Comment:
Let's make sure we replace all links pointing to https://pytorch-lightning.readthedocs.io/en/stable with https://lightning.ai/docs/pytorch/stable and https://lightning.ai/docs/fabric/stable :))